package.json

```diff
@@ -10,8 +10,8 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.2",
     "@fortawesome/free-solid-svg-icons": "^5.2.0",
     "@fortawesome/react-fontawesome": "^0.1.0",
-    "@gen3/guppy": "git+https://github.com/uc-cdis/guppy.git",
-    "@gen3/ui-component": "0.2.2",
+    "@gen3/guppy": "git+https://github.com/uc-cdis/guppy.git#feat/locked-fields",
+    "@gen3/ui-component": "git+https://github.com/uc-cdis/gen3-ui-component.git#feat/lockable-filter",
```

Comment:
I don't think we should merge branches into master....

Reply:
Yeah, I'll wait for guppy and ui-component's PR to be merged and tagged. After that I'll update this package.json